bigquery_etl/glean_usage/templates/cross_channel.view.sql

```diff
@@ -0,0 +1,11 @@
+-- Generated via ./bqetl glean_usage generate
+CREATE OR REPLACE VIEW
+    `{{ project_id }}.{{ target_view }}`
+AS
+{% for (dataset, channel) in datasets -%}
+{% if not loop.first -%}
+UNION ALL
+{% endif -%}
+SELECT * REPLACE("{{ channel }}" AS normalized_channel)
```

Comment:
I forgot that for Fenix, this channel thing is complicated enough to require a UDF.

So we should have logic here for `if app_name = "fenix"` and this line would be:

```
 SELECT * REPLACE(mozfun.norm.fenix_app_info("{{ dataset }}", app_build_id).channel AS normalized_channel)
```